sections/1-introduction.tex

```diff
@@ -1,101 +1,76 @@
 The National Academies of Sciences, Engineering, and Medicine's 2018 report on Open Source Software Policy Options for NASA Earth and Space Sciences \citep{NAP2018} outlines several solutions to alleviate this problem -- namely that the NASA Science Mission Directorate provide funding for new and existing open source software projects, promote scientists who spend time developing and improving open source software projects, and offer prizes for exemplary contributions to the open source software community. 
 The \sunpy community supports solutions like these for all relevant funding agencies and furthermore has the ability to accept financial contributions from institutions or individuals through the NumFOCUS organization.
 
-\section{Development Model - NABIL}
-% edited by Steven C. 23-Apr-2019
+\section{Development Model}
 
-To satisfy the mission statement of the project an open development model was adopted for all development. 
+To satisfy the mission statement of the Project, the \sunpy community adopted an open development model for all development. 
 This development model is widely used within the scientific Python community. 
-The \sunpy organization and \sunpypkg package are hosted on \github and use Git (\url{https://git-scm.com/}) as its distributed version control software.
+The \sunpy organization and \sunpypkg package are hosted on \github and use Git\footnote{\url{https://git-scm.com/}} as its distributed version control software.
 The entire codebase is publicly available and anyone can suggest changes through pull requests. 
-Furthermore, the codebase is licensed under a permissive 2-clause BSD licence (\url{https://opensource.org/licenses/BSD-2-Clause}), therefore anyone can redistribute, improve, repackage or use it in a closed environment as long as the copyright notice and the license disclaimer about the warranty are maintained. 
-In order to maintain a high quality code any contribution much satisfy the following requirements.
+Since the codebase is licensed under a permissive 2-clause BSD licence\footnote{\url{https://opensource.org/licenses/BSD-2-Clause}}, anyone can redistribute, improve, repackage or use it in a closed environment as long as they maintain the copyright notice and the license disclaimer about the warranty. 
+In order to maintain high quality code, every contribution must satisfy the following requirements:
 \begin{enumerate}
     \item Code and documentation must follow widely used style guides (PEP 8 and numpydoc).
-    \item Documentation must be provided with all new features which includes code comments, formal documentation and gallery examples.
-    \item Test code must be provided with coverage at or above X\%.
+    \item All new features must be accompanied with documentation. 
+    This includes code comments, formal documentation, and gallery examples.
+    \item Test code must be provided with coverage at or above\todo{Figure out this number}X\%.
```

Comment:
We aren't super strict on a % here. We could just drop mention of hard numbers?

Reply:
I dropped the percentage and changed it to the following:

> Test code must be provided with coverage reports.